Makefile

```diff
@@ -510,7 +510,7 @@ helm-list: ## List helm charts, tag format, and versions in YAML format
   echo "    outDir: '${CHART_BUILD_DIR}'" ;\
 
 .PHONY: chart-install-kind
-chart-install-kind: docker-build kind-load chart postgres-install-kind keycloak-install-kind catalog-install-kind ## install the catalog app in a local kind cluster
+chart-install-kind: docker-build kind-load chart postgres-install-kind catalog-install-kind ## install the catalog app in a local kind cluster
```

Comment:
In fact I think this `chart-install-kind` can go too - was from the old component tests. Maybe @adibrastegarnia can comment